File: src/supabase_client.py
import os
from supabase import create_client, Client
from dotenv import load_dotenv

# Carrega as chaves do arquivo .env
load_dotenv()

url: str = os.environ.get("SUPABASE_URL")
key: str = os.environ.get("SUPABASE_ANON_KEY")

# Cria a conexão oficial
supabase: Client = create_client(url, key)

File: src/supabaseClient.js
import { createClient } from '@supabase/supabase-js';

const supabaseUrl = process.env.SUPABASE_URL;
const supabaseKey = process.env.SUPABASE_ANON_KEY;

// Esse objeto 'supabase' é o que usaremos para conversar com o banco
export const supabase = createClient(supabaseUrl, supabaseKey);